kubernetes/deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mad-learning-web
+  labels:
+    app: mad-learning-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mad-learning-web
+  template:
+    metadata:
+      labels:
+        app: mad-learning-web
+    spec:
+      containers:
+      - name: mad-learning-web
+        image: itetechmadacr.azurecr.io/mad-learning-web:latest
```

Comment:
Not sure how this is supposed to work when we try to deploy an older version 😅 .

Reply:
This is where manifest templating solutions come in, such as Helm, but for now just also tagging latest in docker push is fine 👍

Reply:
Alright, we'll go with that! 😄